tests/schemapack_/builtin_transformations/infer_relations/path/test_path_str.py

```diff
@@ -0,0 +1,321 @@
+# Copyright 2021 - 2023 Universität Tübingen, DKFZ, EMBL, and Universität zu Köln
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Test reference utils."""
+
+from contextlib import nullcontext
+from typing import Optional
+
+import pytest
+
+from metldata.builtin_transformations.infer_references.path.path_elements import (
+    ReferencePathElement,
+    ReferencePathElementType,
+)
```

Comment:
This is also using the old classes instead of the new ones.

Reply:
Thanks fixed in latest commit.